medical/security/medical_security.xml

```diff
@@ -7,40 +7,38 @@
     </record>
 
     <record id="group_medical_user" model="res.groups">
-        <field name="name">Medical User</field>
```

Comment:
I think you were alphabetizing these, right? How come this one didn't go below the Assistant and Doctor?